src/denoising/docs/task_description.yaml

```diff
@@ -0,0 +1,34 @@
+info:
+  id: denoising
+  name: Denoising
+  v1_url: openproblems/tasks/denoising/README.md
+  v1_commit: 3fe9251ba906061b6769eed2ac9da0db5f8e26bb
+  description: "Removing noise in sparse single-cell RNA-sequencing count data"
+  full_description: |
```

Comment:
In `label_projection`, I've been using this: https://github.com/openproblems-bio/openproblems-v2/blob/main/src/label_projection/api/task_info.yaml#L1-L6

Is this also ok for you?